libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/utils/MapboxNavigationCameraUtils.kt

```diff
@@ -23,3 +28,38 @@ private fun Double.roundTo(numFractionDigits: Int): Double {
     val factor = 10.0.pow(numFractionDigits.toDouble())
     return (this * factor).roundToInt() / factor
 }
+
+/**
+ * Takes the longest animation in the set (including delay an duration) and scales the duration down to match the duration constraint if it's exceeded.
```

Comment:
```suggestion
 * Takes the longest animation in the set (including delay and duration) and scales the duration down to match the duration constraint if it's exceeded.
```

Reply:
This is an internal doc, so gonna merge without the additional CI cycle.